example/webpack.config.js

```diff
@@ -22,7 +22,8 @@ module.exports = {
   resolve: {
     extensions: ['.js', '.jsx'],
     alias: {
-      'use-fetch-hooks': path.resolve(__dirname, '../src'),
+      react: path.resolve(__dirname, './node_modules/react'),
+      'react-dom': path.resolve(__dirname, './node_modules/react-dom'),
```

Comment:
This ensures that `react` and `react-dom` imports in `use-fetch-hooks` resolve to the `node_modules` in `example`. This is necessary since `use-fetch-hooks` does not install its own versions of these libraries anymore.